apps/dh/api-dh/source/DataHub.WebApi/Modules/MarketParticipant/Actor/ActorNode.cs

```diff
@@ -0,0 +1,93 @@
+// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Energinet.DataHub.WebApi.Clients.MarketParticipant.v1;
+using Energinet.DataHub.WebApi.Modules.MarketParticipant.Actor.Models;
+
+namespace Energinet.DataHub.WebApi.Modules.MarketParticipant.Actor;
+
+public static partial class ActorNode
```

Comment:
suggestion: seems incorrect to call this class `ActorNode`, but then it is just a bunch of DataLoaders (I know the term "node" is a bit unclear, but personally I would expect a "Node" class to contain fields and optionally operations/data loaders. Maybe we could just call it "ActorDataLoaders.cs"?

Reply:
Renamed